src/experimental/tools/mediaCapabilitiesProber/probers/DRMInfos.ts

```diff
@@ -57,13 +62,37 @@ export default function probeDRMInfos(
 
   return eme
     .requestMediaKeySystemAccess(type, [configuration])
-    .then((keySystemAccess) => {
-      result.compatibleConfiguration = keySystemAccess.getConfiguration();
-      const status: [ProberStatus, ICompatibleKeySystem?] = [
-        ProberStatus.Supported,
-        result,
-      ];
-      return status;
+    .then(async (keySystemAccess) => {
+      if (!canRelyOnRequestMediaKeySystemAccess(type)) {
+        try {
+          const mediaKeys = await keySystemAccess.createMediaKeys();
+          const session = mediaKeys.createSession();
+          const initData = generatePlayReadyInitData(DUMMY_PLAY_READY_HEADER);
+          await session.generateRequest("cenc", initData);
```

Comment:
Shouldn't we close the session? Is is supposed to be GCed because the `MediaKeys` is not refered to anymore?

Maybe a call to `close` would be safer no?

Reply:
You are right, I have done it in the 2 places where this logic is done.
I also catch `close()` errors as it will announce the keySystem as Unsupported otherwise.